client/src/components/AuditTrail/AuditTrailTable.jsx

```diff
@@ -0,0 +1,64 @@
+import { Heading, Table, Tbody, Thead, Tr } from '@chakra-ui/react';
+import Loader from 'components/common/Loader.jsx';
+import { StyledTableHeader } from 'components/common/Table.jsx';
+import { Fragment } from 'react';
+import { useQuery } from 'react-query';
+import usePagination from 'hooks/usePagination';
+import { TablePagination } from '@material-ui/core';
+import { getAuditTrail } from 'utils/api/auditTrail.js';
+import AuditTrailTableRow from '../AuditTrail/AuditTrailTableRow.jsx';
+
+const AuditTrailTable = () => {
+  const { handleChangePage, handleChangeRowsPerPage, page, rowsPerPage } = usePagination();
+  const { isLoading, isSuccess, data } = useQuery('audit', getAuditTrail);
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  if (isSuccess && data.data.length === 0) {
+    return (
+      <>
+        <Heading size="xl" textAlign="center" mt={5}>
+          No Logs to Show
+        </Heading>
+      </>
+    );
+  }
+
+  return (
+    <>
+      <Table minWidth="unset" width="100%" variant="striped" colorScheme="light">
+        <Thead>
+          <Tr>
+            <StyledTableHeader>User</StyledTableHeader>
+            <StyledTableHeader>Action</StyledTableHeader>
+            <StyledTableHeader>Date</StyledTableHeader>
+            <StyledTableHeader>More Info</StyledTableHeader>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {isSuccess &&
+            data.data
+              .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+              .map((log, index) => (
+                <Fragment key={index}>
+                  <AuditTrailTableRow log={log} />
+                </Fragment>
```

Comment:
I believe you can move the key prop from `Fragment` to `AuditTrailTableRow` and get rid of the fragment. 

Only edit if you're pushing another commit on this branch tho. (Not a big deal)

Reply:
Ill come back for this, ill yoink karins search bar later on